src/manifest.json

```diff
@@ -5,7 +5,7 @@
   "content_security_policy": "script-src 'self' 'unsafe-eval'; object-src 'self'",
   "content_scripts": [
     {
-      "matches": ["https://wallet.google.com/merchant/pages/*/purchaseorderlist*"],
+      "matches": ["https://payments.google.com/merchant/pages/*/purchaseorderlist*"],

```

Comment:
`wallet` still works as well, and that is what I have bookmarked, so we will need to include both for at least a bit.